air-script/tests/main.rs

```diff
@@ -109,3 +109,20 @@ fn indexed_trace_access() {
     let expected = expect_file!["indexed_trace_access/indexed_trace_access.rs"];
     expected.assert_eq(&generated_air);
 }
+
+#[test]
+fn random_values() {
+    let generated_air = Test::new("tests/random_values/random_values.air".to_string())
+        .transpile()
+        .unwrap();
+
+    let expected = expect_file!["random_values/random_values.rs"];
+    expected.assert_eq(&generated_air);
+
+    let generated_air = Test::new("tests/random_values/random_values_simple.air".to_string())
+        .transpile()
+        .unwrap();
+
+    let expected = expect_file!["random_values/random_values.rs"];
+    expected.assert_eq(&generated_air);
+}
```

Comment:
small nit: I would probably prefer to test the simpler case first (`random_values_simple.air`), and then to give the more complex test a more specific name, like `random_values_bindings.air` or `random_values_named.air` and put it second.